### DISCUSSION TITLE
Emails Received In Spam - Improve Reputation

### DISCUSSION THREAD

**Discussion by vkarpyuk** [modoboa/modoboa#2012] [Q&A]
Hi Guys.

I have installed modoboa server a couple of months ago. Honestly, since then, I have been using it and I love the performance and everything. However, I found that each time I send an email to a new mail, the receiver receives the mail in spam. I have tested it with Google, Exchange, and others. All of the receivers received my mail in the spam folder. Once they move it to the "income folder" it won't send to spam again. That's been a problem for me because one of my clients is a big organization and sometimes I have to copy or send mails to a new user. I wanted to check what can I do to improve the reputation so my mails are not rejected nor sent to spam.

Thanks in advance!

Viktor Karpyuk

**Comment by tonioo** [comment]
@vkarpyuk Hi, do you have any additional information? For example, why are you considered as a spammer until a user tells you're not?
Have you configured SPF/DKIM/DMARC for your domain?

**Comment by vkarpyuk** [comment]
@tonioo thanks for your reply! 

As you can see on the picture I have two domains. The first one I used to create the mail and during installation it has been used to create the Lets Encript Certificate. Later I created another domain intended to be used as my company's mail. What I figured out that when I connect to my IMAP/SMTP, the client would raise a message that I have an issue with the certificate because it isn't matching with the original domain(the one I used to install the mail server). 

![domans-configuration](https://user-images.githubusercontent.com/1640951/94430694-8f1db680-0151-11eb-9cbb-37807ef19abb.png)

**Comment by tonioo** [comment]
The TLS certificate should match the domain name you declared in your MX record.

**Comment by vkarpyuk** [comment]
Hi @tonioo, thanks for your reply! I have reinstalled the certificates so they could match the domain but know it looks like the database service PostgreSQL is failing to start and so potfix and dovecot.

Error log:
2020-10-05 15:49:12.334 UTC [79703] FATAL:  could not access private key file "/etc/ssl/private/ssl-cert-snakeoil.key": Permission denied
2020-10-05 15:49:12.334 UTC [79703] LOG:  database system is shut down
pg_ctl: could not start server


Also here you are the image:
<img width="759" alt="Screen Shot 2020-10-05 at 11 20 25 AM" src="https://user-images.githubusercontent.com/1640951/95111741-2dcf8780-06fd-11eb-90d7-0386e03a90b9.png">

<img width="1792" alt="Screen Shot 2020-10-05 at 11 29 17 AM" src="https://user-images.githubusercontent.com/1640951/95112698-85babe00-06fe-11eb-99ce-8c9666194cd2.png">

**Comment by vkarpyuk** [comment]
Please help, I have my mail server down for almost a day.

**Comment by tonioo** [comment]
@vkarpyuk I think your postgres server is not looking for the right certificate. The snakeoil certificate is a default self-signed one generated at server installation, it's not a really good idea to use it. Did you change this?

**Comment by MrGeneration** [comment]
If `mail.timelogbook.com` is your mail server in question, please also ensure that your reverse DNS record is configured. In your case the IP `164.90.144.12` does not resolve your mail server as PTR. This is also a reason (beside not sending via SMTPs and with DKIM & SPF) why your mail server will be considered spammy.

---

You can find a good configuration help for ssl relatewd stuff (including security) from e.g. mozilla:
https://ssl-config.mozilla.org

They provide Dovecot and Postfix ssl configurations as well which helps to secure the server.

If you don't have a public available ssl certificate yet, consider using letsencrypt. You can either use their certbot or other projects like acme.sh (I prefer acme.sh because of DNS-API options).
